src/vdaf/poplar1.rs

```diff
@@ -1,8 +1,8 @@
 // SPDX-License-Identifier: MPL-2.0
 
-//! Work-in-progress implementation of Poplar1 as specified in [[draft-irtf-cfrg-vdaf-04]].
+//! Implementation of Poplar1 as specified in [[draft-irtf-cfrg-vdaf-05]].
```

Comment:
In this spirit, I think we could also remove `(EXPERIMENTAL)` from the `Verifiable Distributed Aggregation Function` header in `README.md`.

Reply:
Done, I also updated the following paragraph a bit.